build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC for ovs
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip;  \
+    powershell -Command Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ;  \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download, extract OpenSSL and copy libraries to ovs
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    powershell -Command Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\  ;  \
+    rm C:\openssl-1.0.2u-x64_86-win64.zip ; \
+    cp C:/libeay32.dll C:/openvswitch/usr/sbin/ ; \
+    cp C:/ssleay32.dll C:/openvswitch/usr/sbin/ ; \
+    mv C:/libeay32.dll C:/openvswitch/usr/bin/ ; \
+    mv C:/ssleay32.dll C:/openvswitch/usr/bin/ ; \
```

Comment:
I still wonder why we need duplicate files. I assume windows search system directory and local directory for DLLs, have you tested just putting them to system directory?

Reply:
We will reverify it , ideally these files should be in system directory but it was not working with these files being present in system folder, however by just having these dlls in sbin and bin, ovs processes works fine
Although, I agree  it make sense to have only one copy of file and not multiple  and right place for dlls are system folder.

Reply:
@tnqn 
Dlls like
libeay32.dll and  ssleay32.dll needs to be available in sbin and bin both the directories, otherwise ovs processes will not start.

Reply:
We can copy or move the two dlls (libeay32.dll and ssleay32.dll) to Windows system path C:/windows/system32, then there is no need to copy these files to any other bin paths (c:/openvswitch/usr/bin and c:/openvswitch/usr/sbin)

Reply:
Windows system path "C:/windows/system32" seems to be working fine